lib/addressfinder/v1/email/batch_verification.rb

```diff
@@ -0,0 +1,93 @@
+module AddressFinder
+  module V1
+    module Email
+      class BatchVerification
```

Comment:
Have you considered the difficulties in integrating this properly with the current batch mechanism for bulk verification? Would a solution that just uses a thread pool in the existing "bulk" class provide as much benefit as this solution?

Reply:
I will take a look

Reply:
I have taken a look, and either approach will have a similar level of effort. This way, our customers will get benefit from the changes as well.